boot/assets.go

```diff
@@ -447,6 +477,29 @@ type TrustedAssetsUpdateObserver struct {
 	modeenvLocked bool
 }
 
+func (o *TrustedAssetsUpdateObserver) UpdateBootEntry() error {
```

Comment:
It would be nice if this implementation could be connected in some way to that of the `TrustedAssetsInstallObserver` implementation, at least from the `EfiLoadOptionParameters()` call on. Perhaps a common helper which takes `updatedAssets []string` and does the rest?

Reply:
I factorized that by the way.